addons/website_sale_delivery/models/sale_order.py

```diff
@@ -124,66 +86,60 @@ def _get_delivery_methods(self, cr, uid, order, context=None):
                 # RIM: hack to remove in master, because available field should not depend on a SOAP call to external shipping provider
                 # The validation error is used in backend to display errors in fedex config, but should fail silently in frontend
                 _logger.debug("%s removed from e-commerce carrier list. %s" % (carrier.name, e))
-
         return available_carrier_ids
 
-    def _get_errors(self, cr, uid, order, context=None):
-        errors = super(SaleOrder, self)._get_errors(cr, uid, order, context=context)
-        if not self._get_delivery_methods(cr, uid, order, context=context):
+    @api.model
+    def _get_errors(self, order):
```

Comment:
Because the super method is defined in [website_sale](https://github.com/odoo/odoo/blob/master/addons/website_sale/models/sale_order.py#L37) and is called from the the [controller](https://github.com/odoo/odoo/blob/master/addons/website_sale/controllers/main.py#L761) there with recordset as an argument. And this improvement is not supposed to be done during migration.